src/main/java/core/basesyntax/Box.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Box<K, V> {
```

Comment:
```suggestion
public class Pair<K, V> {
```

and lets make this class inner to Storage